### ISSUE TITLE
[Feature Request] Human Animation Delay between evolution actions

### ISSUE THREAD

**Issue by tws101** [NoxxDev/NecroBot#604]
I would like to request an option be added to the config file

"EvolutionHumanAnimationDelay": false

When false nothing happens. the bot behaves just as it does now.

"EvolutionHumanAnimationDelay": true

When true the bot will wait 25 seconds (Plus of Minus 3 seconds) between each evolution.  The bot should also be stopped doing nothing else during this time as well.

***Setting this to true will reduce the number of evolutions you could do during a lucky egg but it brings the bot in line with what a human can do which is about one evolve per 20-30 seconds due to animation delay. Basically it makes the bot more human.

**Comment by NoxxDev** [comment]
Bot using `"DelayBetweenPlayerActions": 5000,`

https://github.com/NoxxDev/NecroBot/blob/master/PoGo.NecroBot.Logic/Tasks/EvolvePokemonTask.cs#L126-L129

**Comment by tws101** [comment]
I am using that...  But does that only affect evolution actions or all actions?

***The idea is to only further delay evolution actions.  NOT ALL PLAYER ACTIONS

Evolutions have a long in game animation making it where a player can only do about 70 per 30 minutes in the best conditions....

**Comment by tws101** [comment]
I looked it up and the wiki says 

DelayBetweenPlayerActions (Value)
Delay in milliseconds between transfers/recycles/evolutions/etc.


***We need this feature because evolutions need a much longer delay in order to appear human.

**Comment by CrazyKenaz** [comment]
Agree. I also found the evolution action only spend 5~10 secs. The normal action should be more than 20 secs. It's a big mistake that make them identify bot easily.

**Comment by BoydTM** [comment]
maybe instead of 25 sec it may separate parameter where user can set this time himself ? ( range would be even better ( f.e. "EvolutionHumanAnimationDelay": 15000 - 25000) assuming that necrobot will be able randomly choose some value within the given range ...

**Comment by tws101** [comment]
BoydTM,

Right that would work just fine as well.

**Comment by BoydTM** [comment]
In order to make the necrobot logic "nianticproof" each type of actions should have separate range of values ... but it is always easy to say and may be harder to implement... And still - we do not know what exactly makes bots so detectable (despite of users error in config of course) so ...

**Comment by Brazuca87** [comment]
I've changed my code to: DelayingUtils.Delay(rnd.Next(22500,24000), 0);
That way I get randomized evolution times, within realistic human timer. Not sure I really need to have that as a different setting, so this works for me :)
